frontend/src/app/testQueue/AoEForm/AoEForm.scss

```diff
@@ -0,0 +1,19 @@
+div.usa-form-group--error {
+  .usa-form-group--error {
+    border-left-width: 0;
+    padding-left: 0;
+
+    @media screen and (min-width: 1024px) {
```

Comment:
⚠️ Is there a way to access `mixins` and other imports in this file? 🤔

Reply:
ideally would love to use `@include at-media("desktop")` here

Reply:
related to https://github.com/CDCgov/prime-simplereport/pull/565#discussion_r561298917